care/facility/api/serializers/patient_consultation.py

```diff
@@ -192,6 +196,34 @@ def validate_bed_number(self, bed_number):
             bed_number = None
         return bed_number
 
+    def get_mews_field(self, consultation):
```

Comment:
Can we compute the mews score from the backend itself instead of sending the data to FE and making FE to compute it?

Reply:
yeah I can, but in frontend there is some checks in mews.jsx to flash a hovering dialog that basically which data is missing in that time range, so for that it's looking more informative that mews score is not getting calculated due to absence which parameter, that's why I'm not calculating in backend

Reply:
Which scheme, schema of backend consultation or frontend consultation

Reply:
@rithviknishad please reply

Reply:
Backend. What is front-end schema?

Reply:
![image](https://github.com/coronasafe/care/assets/91010142/ff975315-1436-478b-ad0b-2e349bcbd052)
![image](https://github.com/coronasafe/care/assets/91010142/f9ef32c9-9ff3-4058-8483-e88372cc42e1)
I annotated it, First I thought to make a serialised class for validation and error checking but by default it was getting validated by dailyround, so I just simply annotated like this, is it okay??